text/profiles/0237-profiles-data-model.md

```diff
@@ -0,0 +1,1067 @@
+message ProfileType {
+  // aggregation_temporality describes if the aggregator reports delta changes
+  // since last report time, or cumulative changes since a fixed start time.
+  AggregationTemporality aggregation_temporality = 1;
+
+  // Profiler sample rate in Hz. This parameter indicates the frequency at which samples are collected, specifically for CPU profiles. Common values are 99 or 100. [Optional].
+  uint64 sample_rate = 2;
+
+  // Index into the string table for the type of the sample. Example values are "cpu", "alloc_objects", "alloc_bytes", "block_contentions". Full list is defined in https://github.com/open-telemetry/semantic-conventions
+  uint32 type_index = 3;
+
+  // Index into the string table for the unit of the sample. Example values are "ms", "ns", "samples", "bytes". Full list is defined in https://github.com/open-telemetry/semantic-conventions
+  uint32 unit_index = 4;
+
+  // List of indices referring to Stacktraces in the Profile's stacktrace table.
+  repeated uint32 stacktrace_indices = 10;
+
+  // List of indices referring to Links in the Profile's link table. Each link corresponds to a Stacktrace in stacktrace_indices list. Length must match stacktrace_indices length. [Optional]
+  repeated uint32 link_indices = 11;
+
+  // List of indices referring to AttributeSets in the Profile's attribute set table. Each attribute set corresponds to a Stacktrace in stacktrace_indices list. Length must match stacktrace_indices length. [Optional]
+  repeated uint32 attribute_set_indices = 12;
+
+  // List of values. Each value corresponds to a Stacktrace in stacktrace_indices list. Length must match stacktrace_indices length.
+  repeated int64 values = 13;
+
+  // List of timestamps. Each timestamp corresponds to a Stacktrace in stacktrace_indices list. Length must match stacktrace_indices length.
+  repeated uint64 timestamps = 14;
+}
+
+// SymbolFidelity enumerates the level of fidelity for symbol information in a profile.
+enum SymbolFidelity {
+  SYMBOL_FIDELITY_UNSPECIFIED = 0; // The symbol fidelity level is unspecified.
+  SYMBOL_FIDELITY_FULL = 1;        // The symbol fidelity is at its fullest level of detail.
+}
+
+// Describes the mapping from a binary to its original source code. These are stored in a lookup table in a Profile. These are referenced by index from other messages.
+message Mapping {
+  // Address at which the binary (or DLL) is loaded into memory.
+  uint64 memory_start = 1;
+  // The limit of the address range occupied by this mapping.
+  uint64 memory_limit = 2;
+  // Offset in the binary that corresponds to the first mapped address.
```

Comment:
I know this is copied from the `pprof` definition, and I'm probably missing something obvious, but I'm confused about the word "first" here. Why not the offset that corresponds to this mapped address?